WORKSPACE

```diff
@@ -42,6 +37,21 @@ load("@bazel_skylib//:workspace.bzl", "bazel_skylib_workspace")
 
 bazel_skylib_workspace()
 
+maybe(
+    http_archive,
+    name = "rules_python",
+    sha256 = "5868e73107a8e85d8f323806e60cad7283f34b32163ea6ff1020cf27abef6036",
+    strip_prefix = "rules_python-0.25.0",
+    url = "https://github.com/bazelbuild/rules_python/releases/download/0.25.0/rules_python-0.25.0.tar.gz",
+)
+
+load("@rules_python//python:repositories.bzl", "python_register_toolchains")
+
+python_register_toolchains(
+    name = "python39",
+    python_version = "3.9",
```

Comment:
Please add `ignore_root_user_error` to this, since our build scripts run inside a Docker container as root; it should be similar to https://github.com/google/xls/commit/eb2fb6d81523b09fbd77b63b88381b1ad6c74850

Reply:
Thank you, that fixed the problem!